form.js

```diff
@@ -0,0 +1,11 @@
+const emailAddress = document.getElementById('myEmail');
+const templateForm = document.getElementById('form');
+const validatorMessage = document.getElementById('validator-message');
+templateForm.addEventListener('submit', (e) => {
+  if (emailAddress.value !== emailAddress.value.toLowerCase()) {
+    e.preventDefault();
+    validatorMessage.textContent = 'Kindly input a valid email address!';
+  } else {
+    validatorMessage.textContent = '';
+  }
+});
```

Comment:
Form data should be stored in the browser and should not be wiped out on reload. Kindly check the requirements and fix the issue. Moreover, After filling the form, and not submitting it, the entered values should be saved in the browser.

Reply:
Thanks but I think you are not getting it, here. 
You are actually commenting on my form.js instead of storage.js.

Form.js cant hold the data because the storage command was not declared there.

Reply:
Please go through this video https://www.loom.com/share/4924f0fc4e33417bbfcc6c39aeb53f57